packages/index-converter/src/index-converter.js

```diff
@@ -232,6 +232,10 @@ module.exports = {
         }
         writer_buffer.push(detectionList4);
 
+        logger.info(fileName + "Updating index name in directory structure");
```

Comment:
The string concatenation should be space separated, else it would be something like : 
```
indexFileUpdating index name in directory structure
```
Also it would be good to end the msg with a `.`?